### DISCUSSION TITLE
Синхронизация пользователей из AD

### DISCUSSION THREAD

**Discussion by Davud77** [totumonline/totum-ru-issues-and-discussions#71] [Есть идея, а что если...]
Есть такое предложение, думаю это будет актуально для больших организаций, использовать учетки для авторизаций из Microsoft AD, или просто с ней синхронизироваться, следить за паролями сотрудников так на много удобнее, для всех приложений одни и те же данные для авторизации (word, autocad, почта, да и сам windows) и возможность назначить время жизни пароля, меняем раз в месяц. 

![image](https://user-images.githubusercontent.com/26651156/152340490-40221431-7f42-4321-81cd-ca8114c2febd.png)

**Comment by ttm-alexeypolunin** [comment]
### Угу —> попробуем реализовать в PRO
